lib/services/channel.js

```diff
@@ -0,0 +1,43 @@
+var Service = require('./service');
+
+/**
+ * Service for interacting with the channel endpoints on the Beam REST API.
+ * @access public
+ * @augments {Service}
+ */
+function ChannelService () {
+    Service.apply(this, arguments);
+}
+
+ChannelService.prototype = new Service();
```

Comment:
Little opinionated but maybe you want to use [this](https://nodejs.org/docs/latest/api/util.html#util_util_inherits_constructor_superconstructor)?